Source/ACE/Entity/WorldObject.cs

```diff
@@ -51,7 +51,7 @@ public virtual Position Location
 
         public WeenieHeaderFlag2 WeenieFlags2 { get; protected set; }
 
-        public UpdatePositionFlag PositionFlag { get; protected set; } = UpdatePositionFlag.Contact;
+        public UpdatePositionFlag PositionFlag { get; set; } = UpdatePositionFlag.Contact;
```

Comment:
Um....  Not a huge fan of this.  This isn't the way to solve the always-falling-other-players bug.